stages/07.extra-scripts/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# kuiper2.0 - Embedded Linux for Analog Devices Products
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+# Author: Larisa Radu <larisa.radu@analog.com>
+
+# Variable EXTRA_SCRIPT should contain the path to the extra script. 
+# The script must be placed inside 'adi-kuiper-gen' directory and the path should be relative to that directory.
+# Example: EXTRA_SCRIPT=stages/07.extra-scripts/examples/extra-script-example.sh and file 'extra-script-example.sh' is placed in stages/07.extra-scripts/examples directory.
+
+if [ ! -z "${EXTRA_SCRIPT}" ]; then
+	# Check if file exists
+	if [ -f "${EXTRA_SCRIPT}" ]; then
+	
```

Comment:
Can you add also a check if extension is ".sh" ? And maybe if the file is executable (x access)?

Reply:
The script will be run using 'bash' command, so there is no need for it to be executable or to have the extension '.sh'.